homework/src/app.js

```diff
@@ -1,11 +1,184 @@
-{
-  /* Your homework goes here */
   
   /* STRINGS */
 
+  console.log(`------ STRINGS -------`);
+  console.log(`\n`);
+
+  const myString = "hello,this,is,a,difficult,to,read,sentence";
+  console.log(myString);
+  console.log(`The length of myString is: ${myString.length}`);
+
+  const myNewString = myString.replace(/,/g , ` `); // g(global) is used to replace all occurances of ',' with ' '
+  console.log(myNewString);
+  console.log(`\n`);
+
+  const str = 'dlroW olleH';
+  console.log(`Old string: ${str}`);
+
+  const arr = str.split(``);
```

Comment:
nit: `arr` is not very descriptive, perhaps a variable name `words` might be better.

Reply:
Oh right, you have a point. I'll be careful to use more meaningful variable names. Thanks.